harness_as_dataframe.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.      table_name = f"{bqddf._CONFIG['DATASET']}.{bqddf._CONFIG['FLATTENED_TABLE']}"
</pre><p>Is this likely to change in the near future?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf_common/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
The underlying dictionary values might change but I don't expect these keys to change